frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,93 @@
+Feature: Image Vulnerability in Project
+    As a user, I would like to see Image Vulnerability in Project
+
+Background:
+    Given user is in Developer perspecitve
+    And user in project namespace "aut-image-vulnerabilty"
+    And quay container security operator installed
+
+@regression
+Scenario: Vulnerability in project overview    
+   Given user is in topology page
+   When user goes to project tab on navigation menu
+   Then user can see vulnerabilities in Status section
+   And vulnerabilities has number associated with it
+
+@regression
+Scenario: Image Vulnerability breakdown pop over in project overview    
+   Given user is in project tab
+   When user clicks on the Image Vulnerabilities
+   Then user can see Image Vulnerability breakdown pop over
+   And user can see Vulnerable Container Images and Fixable Container Images
+   And user can see View all link on bottom of pop over
+
+@regression
+Scenario: Visit Image Manifest Vulnerabilities page from status in project overview    
+   Given user is in project tab
+   When user clicks on the Image Vulnerabilities in Status section
+   And user clicks on the View all link at the bottom of Image Vulnerability breakdown pop over
```

Comment:
```suggestion
   And user clicks on the View All link on Image Vulnerability breakdown pop over
```

Reply:
The UI has the `View all` link it can not be changed

Reply:
Other than "View all", there is one more change I suggested. Can you please updated than one.

Reply:
Updated